po/id.po

```diff
@@ -12,7 +12,7 @@ msgstr ""
 "PO-Revision-Date: 2017-10-02 14:06+0700\n"
 "Last-Translator: Mahyuddin <mhy> <yudi.al@gmail.com>\n"
 "Language-Team: Indonesian (Indonesia) <yudi.al@gmail.com>\n"
-"Language: id_ID\n"
+"Language: id\n"
```

Comment:
Please, don't fix non-merged commits with a new patch. Rather fix the wrong commit.
You can do this using `git rebase`

Cheers,

Reply:
Note that this change has to go in the LINGUAS file too.

Reply:
Oke, i will use new pull-request.

Reply:
You can just push in the same branch (with --force) and the current pull request will get updated. No need to create a new one.

Reply:
Thank's